web/src/utils/dashboard/convertSQLData.ts

```
@@ -34,6 +35,57 @@ import {
+        chartPanelRef,
+        hoveredSeriesState,
+        [resultMetaData.value[i]],
+        { queries: [metadata.queries[i]] },
+      ),
+    );
+  }
+
+  // loop on all options
+  if (options && options[0] && options[0].options) {
+    for (let i = 1; i < options.length; i++) {
+      if (options[i] && options[i].options && options[i].options.series) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)